### ISSUE TITLE
parsed time zone different for fixed and relative dates

### ISSUE THREAD

**Issue by user500** [scrapinghub/dateparser#100]
It appears that `dateparser.parse` converts a fixed date and time to the local time zone and a relative date and time to UTC? I was wondering if it might be possible to add an option to make them both return the same, or to set the time zone in the returned datetime object? I would like to convert the parsed date and time to a Unix time stamp, but I don't know whether the user will enter a fixed or relative date and time. This is with version 0.2.1 on Ubuntu 14.04.2 LTS. Thank you!

**Comment by user500** [comment]
Thinking about it some more, it might be nice if the relative date and times continued to be parsed to UTC and there was an option to have the fixed date and times remain in the time zone parsed from the string (if any, UTC otherwise). In both cases would it be possible to have the time zone information stored in the returned datetime object?

**Comment by waqasshabbir** [comment]
Hi @user500! thank you for taking out time and report this. We are planning to introduce configurations in release 0.5 which should also address this issue. Thanks once again!

**Comment by waqasshabbir** [comment]
Hi @user500, we've introduced `TIMEZONE` setting to dateparser. You can use it as follows. By default, all dates are considered in `UTC`. If timezone setting is specified, dates are converted to the given timezone. 

```
# relative date
In [10]: dateparser.parse('2 hours ago')  # UTC
Out[10]: datetime.datetime(2016, 1, 20, 7, 54, 40, 195695)

In [9]: dateparser.parse('2 hours ago', settings={'TIMEZONE': 'US/Eastern'})
Out[9]: datetime.datetime(2016, 1, 20, 2, 54, 26, 450596)
```

```
# full date
In [17]: dateparser.parse('21 december 2015, 22:10') # assumes UTC
Out[17]: datetime.datetime(2015, 12, 21, 22, 10)

In [18]: dateparser.parse('21 december 2015, 22:10', settings={'TIMEZONE': 'Asia/Karachi'})
Out[18]: datetime.datetime(2015, 12, 22, 3, 10)
```

**Comment by waqasshabbir** [comment]
Fixed in PR #132
